content/people/peymant.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Peyman Tehrani
+role: student-phd
+status: student-phd
+yearjoined: 2017
+portrait: peymant.jpg
+badge: peymant-badge.jpg
+interests:
+  - Resource allocation and optimization algorithms 
+  - Wirelless communications and networks
+  - Machine learning and Reinforcement learning
+  - Internet of things and edge computing
+  - Unmanned aerial vehicles
+education:
+  - degree: M.Sc. in Electerical Engineering-Communication Systems

```

Comment:
Let’s name the degrees consistently: Change ‘M.Sc.’ to ‘M.S.’

Reply:
I changed someone else's from M.S. to M.Sc. (I think damiri).
What do we prefer? Because we now must check across all the profiles.

Reply:
We are sticking to ‘B.S’ and ‘M.S.’

(In the case of Delaram’s page, the [request for change][prev] was also to
change ‘B.Sc.’ and ‘M.Sc.’ to ‘B.S.’ and ‘M.S.’, respectively—not vice versa.)

[prev]: https://github.com/uci-iasl/website/pull/10#discussion_r194211185